### ISSUE TITLE
[rllib] Distributed learning

### ISSUE THREAD

**Issue by richardrl** [ray-project/ray#3923]
Are there any plans to implement something like distributed SGD for pytorch?

The bottleneck for most state space environments in my experience is the speed of backpropagation in the learner not the rate of actor data acquisition

**Comment by robertnishihara** [comment]
Closing due to inactivity. Feel free to reopen.

**Comment by richardrl** [comment]
Hi, yes, MultiGPU with Pytorch. Specifically I am dealing with slow backpropagation through <10,000 parameters in a graph neural network architecture. The bottleneck is not memory, but speed of loss.backwards() 

If it's not implemented, any resources to get it implemented?
